modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -88,27 +116,93 @@ data:
         basicAuth:
           username:
             name: remoteWriteAuth
-            key: user
+            key: <username>
           password:
             name: remoteWriteAuth
-            key: password
+            key: <password>
+----
++
+* bearer token 
++
+A bearer token (`Bearer`) configuration combines a basic authentication username and password with authorization credentials configured to use a specified token (such as an API key).
++
+The following shows a `Bearer` authorization sample that uses a `basicAuth` username and password combined with an authorization token.
+It assumes you have already created a `Bearer` secret with the name `rw-auth` in the `openshift-monitoring` namespace.
+Substitute your username, password, and token for the appropriate `key` values. 
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write.endpoint"
```

Comment:
Not sure, if in this example, we care about the first two entries of this array since only the last entry contains information actually relevant for user wanting to use this method of configuration

Reply:
deleted lines 144 to 152 and changed the descriptive text